src/rprblender/export/image.py

```diff
@@ -76,15 +80,14 @@ def sync(rpr_context, image: bpy.types.Image):
         data = np.flipud(data.reshape(image.size[1], image.size[0], image.channels))
         rpr_image = rpr_context.create_image_data(image_key, np.ascontiguousarray(data))
 
-    rpr_image.set_name(image_key)
+    rpr_image.set_name(image.name + color_space)
```

Comment:
Please use rpr_image.set_name(str(image_key)) to correspond with other exported objects.